community/hangouts/index.md

```diff
@@ -11,20 +11,14 @@ meta_descr: hangouts.intro
     <section class="hangouts-social container">   
         <ul class="row center-xs">
             <li>
-                <a class="ext-noicon" href="https://twitter.com/monero" target="_blank" rel="noreferrer noopener" aria-label="Twitter logo"><div class="social-icon twitter"></div></a>
+                <a class="ext-noicon" href="https://x.com/monero" target="_blank" rel="noreferrer noopener" aria-label="Twitter logo"><div class="social-icon twitter"></div></a>
```

Comment:
Its been requested by a few people on matrix that we remove the monero twitter, as its doing a very poor job of remaining impartial. (and nobody wants to admit to tweeting)